Source/RxCBCentralManager.swift

```diff
@@ -53,7 +53,7 @@ class RxCBCentralManager: RxCentralManagerType {
 
     @objc private class InternalDelegate: NSObject, CBCentralManagerDelegate {
         let didUpdateStateSubject = PublishSubject<BluetoothState>()
-        let willRestoreStateSubject = PublishSubject<[String: Any]>()
+        let willRestoreStateSubject = BehaviorSubject<[String: Any]?>(value: nil)
```

Comment:
Couldn't this be a ReplaySubject instance with buffer size 1? It doesn't need the starting value.

Reply:
Thanks for your review!
I'll try the change and check behavior on my project as soon as possible.

Reply:
@Kacper20 I tried your suggestion and the changes worked well!